### PR TITLE
add pylogit

### PR DESCRIPTION
**Pull request by gboeing**

### CODE REVIEW

File: recipes/pylogit/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
```
Comment (synapticarbors) [inline recipes/pylogit/meta.yaml]: It looks like there are some version restrictions on the dependencies:

https://github.com/timothyb0912/pylogit/blob/master/setup.py#L88-L93

File: recipes/pylogit/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```
Comment (synapticarbors) [inline recipes/pylogit/meta.yaml]: Can this be `noarch: python`?

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

File: recipes/pylogit/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    - tqdm
+
+test:
+  imports:
+    - pylogit
+
+about:
+  home: https://github.com/timothyb0912/pylogit
+  license: BSD-3-Clause
+  license_family: BSD
```
Comment (synapticarbors) [inline recipes/pylogit/meta.yaml]: Please manually package the license file:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

and include a link to the following PR as a comment in the recipe:

https://github.com/timothyb0912/pylogit/pull/18

File: recipes/pylogit/meta.yaml (hunk 2 above)
Comment (gboeing) [reply recipes/pylogit/meta.yaml]: It's not totally clear to me from that doc. This package uses the `future` package for python 2+3 compatibility. I'm not sure what role its dependencies such as pandas etc play in this decision.

File: recipes/pylogit/meta.yaml (hunk 1 above)
Comment (gboeing) [reply recipes/pylogit/meta.yaml]: I have added these.

File: recipes/pylogit/meta.yaml (hunk 3 above)
Comment (gboeing) [reply recipes/pylogit/meta.yaml]: I have added this.

File: recipes/pylogit/meta.yaml (hunk 2 above)
Comment (gboeing) [reply recipes/pylogit/meta.yaml]: I added noarch: python

**Comment by gboeing** [comment]
@synapticarbors this is ready to merge.

**Pull request merged**